### PR TITLE
Adds nightly test for reading BQ table with complex schema

### PR DESCRIPTION
**Pull request by prashastia**
Complex schema table in BQ is created with about 15 levels of nesting (the max allowed).
the e2e test performs a read on the same.

### CODE REVIEW

File: cloudbuild/nightly.sh [hunk 1]
```diff
@@ -32,6 +32,8 @@ case $STEP in
   e2e_bounded_read_tests)
     # 1. Run the simple bounded table test.
     source cloudbuild/e2e-test-scripts/table_read.sh "$PROJECT_ID" "$CLUSTER_NAME_SMALL_TEST" "$REGION_SMALL_TEST" "$PROJECT_NAME" "$DATASET_NAME" "$TABLE_NAME_SIMPLE_TABLE" "$AGG_PROP_NAME_SIMPLE_TABLE" "" "bounded"
+    # 2. Run the complex schema table test.
+    source cloudbuild/e2e-test-scripts/table_read.sh "$PROJECT_ID" "$CLUSTER_NAME_SMALL_TEST" "$REGION_SMALL_TEST" "$PROJECT_NAME" "$DATASET_NAME" "$TABLE_NAME_COMPLEX_SCHEMA_TABLE" "$AGG_PROP_NAME_COMPLEX_SCHEMA_TABLE" "" "bounded"
```
Comment (jayehwhyehentee) [inline cloudbuild/nightly.sh:36]: Better to add each test in its own step.

Comment (prashastia) [reply cloudbuild/nightly.sh]: Added.

**Review by jayehwhyehentee — APPROVED** [review]

**Pull request merged**